python/lsst/pipe/supertask/configOverrides.py

```diff
@@ -62,6 +65,20 @@ def addValueOverride(self, field, value):
         """
         self._overrides += [('value', (field, value))]
 
+    def addNameTemplates(self, nameDictString):
+        """Add keys and values to be used in formatting config nameTemplates
+
+        This method takes in a dictionary passed in on the command line as
+        a string in the format of key:value which will be used to format
+        fields in all of the nameTemplates found in a config object.
```

Comment:
This isn't quite enough for me to guess how to actually provide this argument; and example would be welcome, but probably doesn't actually belong here.  Maybe add an example to the ArgumentParser option, and a some kind of (manual) pointer to it here?